modules/options.nix

```diff
@@ -5,22 +5,38 @@
   ...
 }:
 with lib; let
+  # Added 2024-03-26 (do not remove)
+  legacyAliases = [
+    {
+      from = "options";
+      to = "opts";
+    }
+    {
+      from = "globalOptions";
+      to = "globalOpts";
+    }
+    {
+      from = "localOptions";
+      to = "localOpts";
+    }
+  ];
```

Comment:
Should there be a test added to ensure these aren't accidentally removed? I don't have a clue how to write such a test though...

Reply:
Hmmm I think the comment should be enough